pandas/core/dtypes/common.py

```diff
@@ -1795,7 +1795,10 @@ def _get_dtype(arr_or_dtype):
     if isinstance(arr_or_dtype, np.dtype):
         return arr_or_dtype
     elif isinstance(arr_or_dtype, type):
-        return np.dtype(arr_or_dtype)
+        try:
```

Comment:
what are these changes for?

Reply:
Thought I needed them at some point. Tests seem to pass without them. Reverted.